build.sbt

```diff
@@ -128,14 +128,13 @@ lazy val grpcTests =
 // provides offset storage backed by akka-persistence-r2dbc
 lazy val r2dbc =
   Project(id = "akka-projection-r2dbc", base = file("akka-projection-r2dbc"))
-    .configs(IntegrationTest)
-    // .settings(Scala3.settings) FIXME can be done once r2dbc 1.1.0 is out
+    .configs(IntegrationTest.extend(Test))
+    .settings(Scala3.settings) // FIXME bump dependency once r2dbc 1.1.0 is out
```

Comment:
```suggestion
    .settings(Scala3.settings)
```

Reply:
this is done already, I mean we are on milestone still but that is not related to Scala3